src/js/view/controls/controls.js

```diff
@@ -259,6 +271,22 @@ export default class Controls {
         this.playerContainer.addEventListener('keydown', handleKeydown);
         this.keydownCallback = handleKeydown;
 
+        // If not mobile, show controls after closing overlay
+        if (!OS.mobile) {
+            const blurHandler = (evt) => {
```

Comment:
How does `playerContainer` emitting a "blur" event equate to the overlay being closed?